crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -734,6 +760,7 @@ mod tests {
             BeaconConsensusEngineHandle::new(to_engine),
             payload_store.into(),
             task_executor,
+            client,
```

Comment:
can you add a test like the ones below to check that the endpoint works? e.g. something like

```rs
#[tokio::test]
async fn engine_client_version_v1() {
    let (_, api) = setup_engine_api();
    let res = api.get_client_version_v1().await;
    assert_matches!(res, ...)
}
```